### DISCUSSION TITLE
How to add contour to subtitles (SubtitlesClip)

### DISCUSSION THREAD

**Discussion by samuelbraun04** [Zulko/moviepy#2110] [Q&A]
Is it possible to add contour/outline to subtitles (SubtitlesClip) in moviepy? 

Text with no contour:

![image](https://github.com/Zulko/moviepy/assets/90664891/146c6f4e-1592-46f8-8694-e45057237db1)

Text with contour:

![image](https://github.com/Zulko/moviepy/assets/90664891/82c79bae-6d9e-4601-9955-8a63724a2425)

**Comment by marcus49007** [comment]
yes any body please help
